CMSgov/cms-carts-seds
# [MDCT-2914] 3C - Not readonly & Prior year info

### Description

Feedback from CARTS folks caught some extra changes for 3C.

Part 5 & 6
- Read data from 2022 (was not working in Part 5)
- Current year answers should not be read only in the 2 year cycle
- Functional Change: Answers from prior year should not be readonly, and should be editable.
- Question 9 in part 5 & 6 should also pull through (🐛) 
  - When Question 9 is empty, no string should be populated. Functionality was broken and not happening, but restoring it started bringing a placeholder string through. Removed.

### Related ticket(s)


MDCT-2914

---
### How to test

- Spin up 2022 and 2023. 
- Fill out the grids in part 5 and 6
- Go to 2023. New year should be editable
- Past year should be populated and editable

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
